### ISSUE TITLE
Requesting node-sass module with native Mac ARM or Universal build support

### ISSUE THREAD

**Issue by nayankankariya** [sass/node-sass#3367]
Hi,

This is regarding node-sass module used in our project. We are migrating our application for having native support on Mac ARM platforms and see that node-sass being Intel only binary, we are unable to migrate to Mac ARM due to the dependency on node-sass Intel binary which fails to compile with Node ARM. Hence we request you to help us with node-sass package with native ARM or Universal binary to successfully build our app with native Mac ARM support.

Please let us know in case you are looking for any additional info from our side. Awaiting your kind response on helping us getting unblocked with the behavior where node-sass doesn't work with the ARM version of Node runtime. We do not intend to use Rosetta as we require app to be supported natively on Mac ARM platforms without any dependency on Rosetta.

We see this issue request could be related to the ticket https://github.com/sass/node-sass/issues/3033 which is still in Open state.

Thanks,
Nayan

- NPM version (`npm -v`):
- Node version (`node -v`): 16.13.2
- Node Process (`node -p process.versions`):
- Node Platform (`node -p process.platform`): Mac ARM
- Node architecture (`node -p process.arch`): Mac ARM
- node-sass version (`node -p "require('node-sass').info"`): 7.0.1
- npm node-sass versions (`npm ls node-sass`):

**Comment by nschonni** [comment]
This is tracked by #3033, don't open duplicate tickets please

**Comment by nayankankariya** [comment]
Hi @nschonni,

Thanks for your response. We are unable to add a comment for requesting support in the tracking ticket https://github.com/sass/node-sass/issues/3033, hence we created a new ticket for your kind consideration of our request. It will be helpful if we can know any tentative timeline by when node-sass team could help us with native compatible support on Mac ARM platform.

Have a pleasant day!

Regards,
Nayan

**Comment by nschonni** [comment]
Nothing has moved in terms of GitHub Actions supporting a CI runtime for Apple ARM, so there is no update or tentative timeline from the last discussion when we locked that issue

**Comment by nschonni** [comment]
We have no capacity to self-host runners, which is why that other issue is stalled.
